training-event/src/test/features/TrainingEventSummary.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Retrieve Training Event Summaries
+
+  Scenario: Successfully retrieve trainer summaries
+    Given the training service is operational
+    When I send a GET request to "/event/summary"
+    Then the response status should be 200
+    And the response body should contain a list of trainer summaries
+
+  Scenario: Fail to retrieve trainer summaries when service is unavailable
```

Comment:
How would it be unavailable? What is use-case we're trying to validate if we just throw fake exception and expect 500 Internal Server Error, which will always be by default? This test does not validate any exception handling, retries or circuit breaker, it makes no sense to have it.